.gitignore

```diff
@@ -16,7 +16,7 @@ newrelic_agent.log
 .bower-cache
 .vagrant
 Vagrantfile
-
+TODO
```

Comment:
I've added a small TODO file using https://github.com/aziz/PlainTasks to track simple changes, I can remove it later (adding a note to description)